main.go

```diff
@@ -20,7 +20,6 @@
 package main
```

Comment:
I think we can just assume v1 is available, since it's the only existing version in the K8s version we're targeting. End of the day I believe we can remove everything discovery-related, wdyt?